src/styles/calendar.scss

```diff
@@ -0,0 +1,52 @@
+
+@import "./variables.scss";
+
+.calendar {
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  max-width: $container-width;
+  padding: 30px;
+  gap: $block-gap;
+
+  &__block {
+    height: $block-part - 2 * $block-gap;
+    width: $block-part - 2 * $block-gap;
```

Comment:
Your gap is the same as your border so it works, you can add box-sizing: border-box to calendar__block and then you'll be able to left $block-part only

Reply:
Jees, thought reset cnd had box-sizing! (: